api/src/main/java/org/pmiops/workbench/db/jdbc/ReportingQueryServiceImpl.java

```diff
@@ -462,10 +463,14 @@ public List<ReportingWorkspace> getWorkspaces(long limit, long offset) {
                 + "FROM workspace w\n"
                 + "  JOIN cdr_version c ON w.cdr_version_id = c.cdr_version_id\n"
                 + "  JOIN access_tier a ON c.access_tier = a.access_tier_id\n"
+                + "WHERE active_status = "
+                + WorkspaceActiveStatus.ACTIVE.ordinal()
```

Comment:
our usual pattern here is `workspaceActiveStatusToStorage()` - because the values may not necessary be equal to the ordinals.  consider a case where we remove one of the options.

Reply:
Good call! I'll fix that. Thanks!